tests/stub/routing/test_routing_v5x0.py

```diff
@@ -0,0 +1,2857 @@
+from collections import defaultdict
+import time
+
+from nutkit.frontend import Driver
+import nutkit.protocol as types
+from tests.shared import (
+    driver_feature,
+    get_dns_resolved_server_address,
+    get_driver_name,
+    get_ip_addresses,
+)
+
+from ._routing import RoutingBase
+
+
+class RoutingV5x0(RoutingBase):
```

Comment:
I assume you copied and adopted the tests from `RoutingV4x4`. That's fine but I'd like to see `RoutingV4x4` be a child class of `RoutingV5x0` which changes only the necessary methods and explicitly defines all other methods and make them delegate to `RoutingV5x0`. Have a look at `RoutingV4x3` for an example.

Reply:
Do I really need to redeclare all the test methods even though they just call super? That's a bit strange to me.

Reply:
Done anyway :)